Integrations/integration-Carbon_Black_Enterprise_Live_Response.yml

```diff
@@ -628,7 +628,7 @@ script:
     }
 
     function createCommandAndWait() {
-        let result = createCommandRequest(args.session, args.name, args.timeout, args.object, args.compress, args.wait,
+        let result = createCommandRequest(args.session, args.name, args.timeout, args.object, args.commpress, args.wait,
```

Comment:
Why the changes from `commpress` to `compress`? (I know that `commpress` is not the correct spelling of the word but it's what is used in Carbon Black's API)

Reply:
Fixed the typo in latest commit. Thanks